python/ray/experimental/serve/constants.py

```diff
@@ -15,3 +15,6 @@
 
 #: HTTP Port
 DEFAULT_HTTP_PORT = 8000
+
+#: Default path for profiling services
+SERVE_PROFILE_PATH = "/RayServeProfile"
```

Comment:
does this need to be a constant? why can't we just make all endpoints profile-able?

Reply:
All endpoints are profile-able: